crypto/aes/aes_ige.c

```diff
@@ -38,15 +38,30 @@ void AES_ige_encrypt(const unsigned char *in, unsigned char *out,
                      size_t length, const AES_KEY *key,
                      unsigned char *ivec, const int enc)
 {
+    /* Ignore the return value since we are void */
+    if (!ossl_assert(AES_ige_encrypt_ex(in, out, length, key, ivec, enc)))
+        return;
+}
+
+int AES_ige_encrypt_ex(const unsigned char *in, unsigned char *out,
```

Comment:
I'm sorry to say this, but I don't find this appropriate. Though for reason different from suggestion to get rid of asserts. It's about IGE itself. Problem with it is that it's hard-wired to AES_[en|de]crypt with no option to use side-channel-resistant alternatives. Right thing to do about IGE is to *scrap* current implementation and replace it with one integrated into EVP, not to provide additional interface [that is considered an improvement]. In other words I would like to suggest to abstain from modifications to IGE [and effectively declare interface deprecated].